src/design-system/components/Box/Box.tsx

```diff
@@ -58,9 +58,9 @@ export const Box = forwardRef(function Box(
   },
   ref,
 ) {
-    let hasBoxStyles = false;
```

Comment:
don't really care, but curious what causing the formatting diff here

Reply:
I think a few style issues were introduced here. Prettier ran on the git commit that resolved them, I guess. https://github.com/rainbow-me/browser-extension/pull/1180